commands/registry.go

```diff
@@ -140,54 +146,57 @@ func Repository() *Command {
 			Use:     "repository",
 			Aliases: []string{"repo", "r"},
 			Short:   "Display commands for working with repositories in a container registry",
-			Long:    "The subcommands of `doctl registry repository` help you command actions related to a repository.",
+			Long:    "The subcommands of `doctl registry repository` allow you to manage various properties of your repository.",
 		},
 	}
 
-	listRepositoriesDesc := `This command retrieves information about repositories in a registry, including:
+	listRepositoriesDesc := `Retrieves information about repositories in a registry, including:
   - The repository name
-  - The latest tag of the repository
+  - The latest tag for the repository
   - The compressed size for the latest tag
   - The manifest digest for the latest tag
   - The last updated timestamp
 `
-	CmdBuilder(
+	cmdListRepositories := CmdBuilder(
 		cmd,
 		RunListRepositories, "list",
 		"List repositories for a container registry", listRepositoriesDesc,
 		Writer, aliasOpt("ls"), displayerType(&displayers.Repository{}),
 		hiddenCmd(),
 	)
+	cmdListRepositories.Example = `The following example lists repositories in a registry named ` + "`" + `example-registry` + "`" + ` and uses the ` + "`" + `--format` + "`" + ` flag to return only the digest and update time of each repository: doctl registry repository list example-registry --format Digest,UpdatedAt`
 
-	listRepositoriesV2Desc := `This command retrieves information about repositories in a registry, including:
+	listRepositoriesV2Desc := `Retrieves information about repositories in a registry, including:
   - The repository name
   - The latest manifest of the repository
   - The latest manifest's latest tag, if any
   - The number of tags in the repository
   - The number of manifests in the repository
 `
 
-	CmdBuilder(
+	cmdListRepositoriesV2 := CmdBuilder(
 		cmd,
 		RunListRepositoriesV2, "list-v2",
 		"List repositories for a container registry", listRepositoriesV2Desc,
 		Writer, aliasOpt("ls2"), displayerType(&displayers.Repository{}),
 	)
+	cmdListRepositoriesV2.Example = `The following example lists repositories in a registry named ` + "`" + `example-registry` + "`" + ` and uses the ` + "`" + `--format` + "`" + ` flag to return only the digest and update time of each repository: doctl registry repository list-v2 example-registry --format Digest,UpdatedAt`
```

Comment:
```
$ doctl registry repository list-v2 asb --format Digest,UpdatedAt
Error: unknown column "Digest"
```

`Digest` is not valid here, only `Name`, `LatestManifest`, `LatestTag`, `TagCount`, `ManifestCount`, and `UpdatedAt`